01week/rockPaperScissors.js

```diff
@@ -37,8 +38,16 @@ console.log('Hand 1 is: ' + h1 + ' '  + 'and Hand 2 is: ' + h2);
   }
   else if (h1==="scissors"&&h2==="rock"){
     console.log("Hand 2 wins!")
-  }
+  }*/
 
+  if(h1==="scissors" && h2==="paper"|| h1==="paper" && h2==="rock"|| h1==="rock" && h2==="scissors"){
+    console.log("Hand one wins!")
+    return "Hand one wins!"
+  }else if (h1==="paper" && h2==="scissors"||h1==="rock" && h2==="paper" || h1==="scissors" && h2==="rock"){
+    console.log("Hand two wins!")
+    return "Hand two wins!"
+  }
+  
```

Comment:
Great logic! Can you think of a way to check if the user actually input only "rock" "paper" or "scissors". If they input anything else, nicely ask them to redo it?

Reply:
I will see what I can do!  Thank you!